### DISCUSSION TITLE
404 Guide page on Wiki.js' wiki

### DISCUSSION THREAD

**Discussion by LordThi** [requarks/wiki#7290] [Error / Bug Report]
Hi everyone,
During my learning of wiki.js, I wanted to test your breadcrumb. I was on https://docs.requarks.io/en/guide/structure page but it doesn/t maker. I clicked on the guide link wich is right after the home link ![image](https://github.com/user-attachments/assets/cd0270d3-17af-4358-a051-5d1e7cab7c91), and i found a 404 (https://docs.requarks.io/en/guide).

**Comment by fnafox** [comment]
This is expected behavior. Not all intermediate pages exist in this context. I suggest familiarizing yourself with wikijs structure which is explained on that very [page](https://docs.requarks.io/en/guide/structure).

**Reply by LordThi** [reply]
thank you for your answer. I know what you mean, but maybe, if there no intermediate page, the link don't have to be active? for ux purpose.
